Mathlib/Data/Finsupp/Interval.lean

```diff
@@ -76,9 +75,9 @@ end RangeIcc
 
 section PartialOrder
 
-variable [PartialOrder α] [Zero α] [LocallyFiniteOrder α] (f g : ι →₀ α)
+variable [PartialOrder α] [Zero α] [LocallyFiniteOrder α] [DecidableEq ι] [DecidableEq α]
```

Comment:
But now you are also adding `DecidableEq` for the coefficients...

Reply:
Yes, but only for Finsupp. For MvPowerSeries the type in question is `Nat`, which satisfies this for free.

Reply:
Ok, fair enough